app/services/users/delete_activity.rb

```diff
@@ -5,41 +5,26 @@ module DeleteActivity
     # If you're removing data that is in Elasticsearch, make sure to use
     # .destroy_all to trigger the callback to remove the document(s)
     def call(user)
-      user.notifications.delete_all
-      user.reactions.delete_all
-      user.follows.delete_all
-      Follow.followable_user(user.id).delete_all
-      user.messages.delete_all
-      Users::CleanupChatChannels.call(user)
-      user.mentions.delete_all
-      user.badge_achievements.delete_all
-      user.github_repos.delete_all
-
+      handle_social_media(user)
+      handle_profile_info(user)
```

Comment:
is a more descriptive function name not `delete_profile_info` instead of `handle_profile_info`?

Reply:
and the same for social media?

Reply:
@Ridhwana I agree with you. I will change

Reply:
Thanks @luchiago  😊 Once that's done this looks really good! 🎉 Thanks.